prepare_pacta_indices.R

```diff
@@ -26,7 +26,22 @@ setwd(transition_monitor_dir)
 working_dir <- file.path(transition_monitor_dir, "working_dir")
 
 input_dir <- file.path("/pacta-data/", pacta_financial_timestamp)
-output_dir <- input_dir
+
+output_dir <- file.path("/outputs")
+stopifnot(dir.exists(output_dir))
+
+system_timestamp <- format(
+  Sys.time(),
+  format = "%Y%m%dT%H%M%SZ",
+  tz = "UTC"
+)
+output_dir <- file.path(output_dir, paste(Sys.getenv("R_CONFIG_ACTIVE"), system_timestamp, sep = "_"))
```

Comment:
```suggestion
output_dir <- file.path(output_dir, paste(pacta_financial_timestamp, system_timestamp, sep = "_"))
```

This gives us the ability to add config options like `2023_azure` (matching what we have in `workflow.data.preparation`)